airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -357,7 +357,7 @@ def post_dag_run(*, dag_id: str, session: Session = NEW_SESSION) -> APIResponse:
             )
             dag_run_note = post_body.get("note")
             if dag_run_note:
-                current_user_id = getattr(current_user, "id", None)
+                current_user_id = get_auth_manager().get_user().get_id()
```

Comment:
I wonder if the interface should be `get_auth_manager().get_user_id()` instead. In many situations the ID can be retrieved very cheaply without building a concrete user object.

Reply:
Sounds like a good point. Let me do that